code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -1,3 +1,4 @@
+#define COLOSSUS_SLEEP(X) sleep(X); if(QDELETED(src)) return;
```

Comment:
why does the colly not check for stat but the dragon does

Reply:
Colossus deletes on death, drake leaves a corpse